### PR TITLE
fix ncat sctp test and update CI image 

### PR DESCRIPTION
**Pull request by Luap99**
see commits

### CODE REVIEW

**Comment by Luap99** [comment]
@edsantiago I hope this unblock us hopefully

**Comment by mheon** [comment]
LGTM once it passes CI

**Comment by Luap99** [comment]
@edsantiago @mheon PTAL again

File: test/helpers.bash [hunk 1]
```diff
@@ -644,7 +645,7 @@ function run_nc_test() {
     fi
 
     nsenter -n -t "${CONTAINER_NS_PIDS[$container_ns]}" timeout --foreground -v --kill=10 5 \
-        nc $nc_common_args -l -p $container_port &>"$NETAVARK_TMPDIR/nc-out" <$stdin &
+        nc $nc_common_args -l -p $container_port &>"$NETAVARK_TMPDIR/nc-out" <&$stdin &
```
Comment (edsantiago) [inline test/helpers.bash:648]: Although CI is passing, it's not for the reasons described in your commit: `$stdin` is, at this point, undefined. I expected bash to treat this as an error, but it doesn't, and I'm completely lost as to what exactly is happening.

I *think* the answer here is to remove `<&$stdin` entirely, given that you've set up the `exec`, but I'm sorry I can't approve this as it is.

**Review by edsantiago — CHANGES_REQUESTED** [review]
Very weird behavior, see inline. 

And your third commit (clippy) doesn't say what the clippy warnings actually are, and I'm assuming that "ipva" is a typo for "ipv6" maybe? Not a blocker from my perspective but I mention it for you to decide whether a future maintainer will care about that or not.

File: test/helpers.bash (hunk 1 above)
Comment (Luap99) [reply test/helpers.bash:648]: sorry I don't follow why do you think $stdin is undefined?

Comment (Luap99) [reply test/helpers.bash:648]: I will try to remove it, if that makes it I happily do that as it makes the test code simpler

Comment (edsantiago) [reply test/helpers.bash:648]: no wait hold on; I'm refreshing my memory on `{varname}`

**Review by edsantiago — APPROVED** [review]

**Comment by edsantiago** [comment]
My bad.

/lgtm

**Pull request merged**